zerver/tests/test_upload.py

```diff
@@ -1305,8 +1305,7 @@ class S3Test(ZulipTestCase):
 
     @use_s3_backend
     def test_file_upload_s3(self) -> None:
-        conn = S3Connection(settings.S3_KEY, settings.S3_SECRET_KEY)
-        bucket = conn.create_bucket(settings.S3_AUTH_UPLOADS_BUCKET)
+        bucket, = create_s3_bucket(settings.S3_AUTH_UPLOADS_BUCKET)
```

Comment:
stylistically, I think `bucket = create_s3_bucket(settings.S3_AUTH_UPLOADS_BUCKET)[0]` is clearer and more standard.